src/main/java/org/gitlab4j/api/PipelineApi.java

```diff
@@ -546,6 +546,11 @@ public PipelineSchedule takeOwnershipPipelineSchedule(Object projectIdOrPath, Lo
         return (response.readEntity(PipelineSchedule.class));
     }
 
+    public void playPipelineSchedule(Object projectIdOrPath, Long pipelineScheduleId) throws GitLabApiException {
```

Comment:
Please add a javadoc to this method. Some was suggested in https://github.com/gitlab4j/gitlab4j-api/issues/906#issue-1571525025 maybe it is a good fit?